x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/summary_actions/index.tsx

```diff
@@ -71,9 +71,13 @@ export const getAllMarkdownCommentsFromResults = ({
     stats: patternRollup.stats,
   });
 
+  let displayDocSize = false;
+
   const summaryTableMarkdownRows: string[] = summaryTableItems.map((item) => {
     const result: DataQualityCheckResult | undefined =
       patternRollup.results != null ? patternRollup.results[item.indexName] : undefined;
+    const sizeInBytes = getSizeInBytes({ indexName: item.indexName, stats: patternRollup.stats });
+    displayDocSize = Number.isInteger(sizeInBytes);
```

Comment:
This logic having a `let` that we update on every iteration of the `map` is a bit weird. This is the same as calling `isInteger` only on the last element of the array.

But I wonder why we need this flag, isn't checking `isILMAvailable` enough?